app/assets/javascripts/application.js

```diff
@@ -12,7 +12,7 @@
 //
 //= require jquery
 //= require jquery-ujs
-//= require popper.js/
+//= require @popperjs/core/dist/umd/popper
```

Comment:
```suggestion
//= require @popperjs/core
```

I think the default cjs version is fine to import

Reply:
I'm really not that good at Javascript, but:

```
Uncaught ReferenceError: exports is not defined
```